circuit-benchmarks/src/super_circuit.rs

```diff
@@ -0,0 +1,115 @@
+//! SuperCircuit circuit benchmarks
+
+#[cfg(test)]
+mod tests {
+    use crate::bench_params::DEGREE;
+    use ark_std::{end_timer, start_timer};
+    use eth_types::{address, bytecode, geth_types::GethData, Word};
+    use ethers_signers::{LocalWallet, Signer};
+    use halo2_proofs::plonk::{create_proof, keygen_pk, keygen_vk, verify_proof, SingleVerifier};
+    use halo2_proofs::{
+        pairing::bn256::{Bn256, G1Affine},
+        poly::commitment::{Params, ParamsVerifier},
+        transcript::{Blake2bRead, Blake2bWrite, Challenge255},
+    };
+    use mock::{TestContext, MOCK_CHAIN_ID};
+    use rand::SeedableRng;
+    use rand_chacha::ChaChaRng;
+    use std::collections::HashMap;
+    use zkevm_circuits::super_circuit::SuperCircuit;
+
+    use halo2_proofs::pairing::bn256::Fr;
+
+    #[cfg_attr(not(feature = "benches"), ignore)]
+    #[test]
+    fn bench_super_circuit_prover() {
+        let mut rng = ChaChaRng::seed_from_u64(2);
+
+        let chain_id = (*MOCK_CHAIN_ID).as_u64();
+
+        let bytecode = bytecode! {
+            STOP
+        };
+
+        let wallet_a = LocalWallet::new(&mut rng).with_chain_id(chain_id);
+
+        let addr_a = wallet_a.address();
+        let addr_b = address!("0x000000000000000000000000000000000000BBBB");
+
+        let mut wallets = HashMap::new();
+        wallets.insert(wallet_a.address(), wallet_a);
+
+        let mut block: GethData = TestContext::<2, 1>::new(
+            None,
+            |accs| {
+                accs[0]
+                    .address(addr_b)
+                    .balance(Word::from(1u64 << 20))
+                    .code(bytecode);
+                accs[1].address(addr_a).balance(Word::from(1u64 << 20));
+            },
+            |mut txs, accs| {
+                txs[0]
+                    .from(accs[1].address)
+                    .to(accs[0].address)
+                    .gas(Word::from(1_000_000u64));
+            },
+            |block, _tx| block.number(0xcafeu64),
+        )
+        .unwrap()
+        .into();
+
+        block.sign(&wallets);
+
+        let (k, circuit, instance) =
```

Comment:
The `k` variable is only used in the print, and afterwards the `DEGREE` variable is used.  I think we should remove this `k` to avoid confusion.